docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -90,6 +91,15 @@ C:\> where ocaml
 C:\Program Files\ocpwin64\4.02.1+ocp1-msvc64-20160113\bin\ocaml.exe
 ```
 
+Openssl for Windows
+-------------------
+Openssl is required to generate certicates used for testing.
```

Comment:
I would move this as a subsection of the Git install to check that things have been appropriately installed. We should also add a troubleshooting section for ensuring that the `where` results point to the `Git\mingw64\bin` versions of bash etc. when WSL is also installed.

Reply:
Currently, we are not supporting WSL. I removed this section and added a relevant line in the guide.